_inc/client/plans/index.jsx

```diff
@@ -12,35 +13,204 @@ import PlanGrid from './plan-grid';
+					title: __( 'Jetpack Backup {{em}}Daily{{/em}}', {
+						components: {
+							em: <em />,
+						},
+					} ),
+					subtitle: __( 'Included in your {{planLink}}Premium Plan{{/planLink}}', {
+						components: {
+							planLink: <a href={ `/plans/my-plan/${ siteRawlUrl }` } />,
+						},
+					} ),
+					description: __( 'Always-on backups ensure you never lose your site.' ),
+					purchase,
+					isCurrent: true,
+				};
+			case 'is-business-plan':
+				return {
+					title: __( 'Jetpack Backup {{em}}Real-Time{{/em}}', {
+						components: {
+							em: <em />,
+						},
+					} ),
+					subtitle: __( 'Included in your {{planLink}}Professional Plan{{/planLink}}', {
+						components: {
+							planLink: <a href={ `/plans/my-plan/${ siteRawlUrl }` } />,
+						},
+					} ),
+					description: __(
+						'Always-on backups ensure you never lose your site. Your changes are saved as you edit and you have unlimited backup archives.'
+					),
+					purchase,
+					isCurrent: true,
+				};
+		}
+	}
+
+	getTitleSection() {
+		return (
+			<Fragment>
+				<h1 className="plans-section__header">{ __( 'Solutions' ) }</h1>
+				<h2 className="plans-section__subheader">
+					{ __( "Just looking for backups? We've got you covered." ) }
+				</h2>
+			</Fragment>
+		);
+	}
+
+	findPrioritizedPurchase() {
+		const { activeSitePurchases } = this.props;
+
+		// Note: the order here is important, as it resolves cases where a site
+		// has both a plan and a product at the same time.
+		const planClasses = [
+			'is-business-plan',
+			'is-realtime-backup-plan',
+			'is-premium-plan',
+			'is-personal-plan',
+			'is-daily-backup-plan',
+		];
+
+		for ( const planClass of planClasses ) {
+			const purchase = activeSitePurchases.find(
+				item => getPlanClass( item.product_slug ) === planClass
+			);
+			if ( undefined !== purchase ) {
+				return purchase;
+			}
+		}
+
+		return false;
+	}
+
 	render() {
 		const {
 			dailyBackupUpgradeUrl,
+			isFetchingData,
 			products,
 			realtimeBackupUpgradeUrl,
-			isFetchingData,
+			sitePlan,
 		} = this.props;
 
-		const plan = get( this.props.sitePlan, 'product_slug' );
+		const plan = get( sitePlan, 'product_slug' );
 		const upgradeLinks = {
 			daily: dailyBackupUpgradeUrl,
 			'real-time': realtimeBackupUpgradeUrl,
 		};
 
+		let singleProductContent;
+
+		const purchase = this.findPrioritizedPurchase();
+		if ( purchase ) {
+			const productCardProps = this.getProductCardPropsForPurchase( purchase );
+			singleProductContent = (
+				<Fragment>
+					{ this.getTitleSection() }
+					<div className="plans-section__single-product">
+						<ProductCard { ...productCardProps } />
+					</div>
+				</Fragment>
+			);
+		} else {
+			singleProductContent = (
+				<Fragment>
+					{ this.getTitleSection() }
+					<SingleProductBackup
+						plan={ plan }
+						products={ products }
+						upgradeLinks={ upgradeLinks }
+						isFetchingData={ isFetchingData }
+					/>
+				</Fragment>
+			);
+		}
+
```

Comment:
I've got two questions here:

1. How about moving this whole block of code to the `SingleProductBackup` so that the Jetpack Backup is contained within a single component?

2. In Calypso we're using `ProductCard` for both pre- and post-purchase states. Could we do the same here? Definitely not a blocker, though.

Reply:
Good suggestion, I think all those components can be simplified and modernized. Let's keep that idea in our box and iterate further soon. 

I suggest going ahead as-is so we can allow this to be tested with the beta that comes out tomorrow.